kotlin-runtime/src/main/kotlin/io/micronaut/kotlin/converters/FlowConverterRegistrar.java

```diff
@@ -34,7 +34,7 @@
 @Requires(classes = {Flux.class, ReactiveFlowKt.class})
 public class FlowConverterRegistrar implements TypeConverterRegistrar {
     @Override
-    public void register(ConversionService<?> conversionService) {
+    public void register(MutableConversionService conversionService) {
```

Comment:
```suggestion
    public void register(ConversionService conversionService) {
```

Reply:
This won't compile. `addConverter()` is in the `MutableConversionService` interface